src/com/projectkorra/projectkorra/configuration/ConfigManager.java

```diff
@@ -943,11 +943,13 @@ public static void configCheck(final ConfigType type) {
 			config.addDefault("Abilities.Air.AirSweep.Damage", 3);
 			config.addDefault("Abilities.Air.AirSweep.Knockback", 3.5);
 			config.addDefault("Abilities.Air.AirSweep.Cooldown", 6000);
+			config.addDefault("Abilities.Air.AirSweep.HitRadius", 1);

```

Comment:
Nice

Reply:
Nice

Reply:
We typically just have this option called 'Radius' in our other moves. Might be worth changing it to that just for consistency's sake.